.circleci/config.yml

```diff
@@ -39,10 +39,12 @@ jobs:
           command: |
             docker-compose --file compose/base.yml --file compose/production.yml up -d
             ./compose/wait-for-it.sh localhost:8000 -t 15
-            docker-compose --file compose/base.yml --file compose/production.yml exec production_app ./manage.py migrate
+      # Despite the webserver working on 8000, the web app still requires a few secs until is ready to show /healthz
```

Comment:
Could we handle this in a more elegant fashion by running ``curl`` in a loop with a ``sleep 1`` and early-outing if we succeed?